### DISCUSSION TITLE
Enhancement: check task in Obsidian when sync'd to TickTick

### DISCUSSION THREAD

**Discussion by ms3056** [thesamim/TickTickSync#109] [Ideas]
Hi - I would like to create tasks in my notes as they occur - but as soon as they are sync'd to TickTick I would like the tasks in Obsidian to be checked as done. I truly want a one-way sync Obsidian → TickTick. 

Thank you for making this plugin.

**Comment by thesamim** [comment]
If tasks are marked done in Obsidian, they are marked done in TickTick. Is that really what you want?

**Comment by LemurTech** [comment]
I don't understand the desired workflow, here. @ms3056 you want the tasks in Obsidian to be marked 'done' as soon as they sync?

**Comment by ms3056** [comment]
Yes - exactly. The workflow is capture in Obsidian and manage in TickTick. As soon as the task is synchronized to TickTick then cross it off in Obaidian. This is a "one way" sync for me. 

I have seen other solutions that do this. I manage ALL tasks in TickTick - I just want to capture some of them in Obsidian - I'll never look at the tasks in Obsidian again.

**Comment by thesamim** [comment]
@ms3056: I've changed this into a discussion because I'd like to get input from other users.

This feels like a very unique workflow. 

The architectural change would be significant. 

If it's a workflow that many users want, I will consider it.

**Reply by ms3056** [reply]
Thanks - I appreciate it should be an "option' if at all. 

If no one else is interested - no worries :)

**Reply by dwrinn** [reply]
I would be interested.
